lib/bolt/transport/ssh/connection.rb

```diff
@@ -261,6 +261,7 @@ def execute(command, sudoable: false, **options)
         end
 
         def copy_file(source, destination)
+          @logger.debug { "Uploading #{source}, to #{destination}" }
```

Comment:
Also `269:        def write_executable_from_content(dest, content, filename)`

Reply:
Should this just be `writting wrapper script to #{destination}`? I kind of held back from logging this because it is something we do behind the scenes.

Reply:
unless I can finid what those methods are for i'm going to delete them.

Reply:
I think net-scp is smart enough to know that the source is a stringIO. Though i guess I wont want to log that.